src/Altinn.AccessManagement.Integration/Clients/PartiesClient.cs

```diff
@@ -12,6 +13,7 @@ namespace Altinn.AccessManagement.Integration.Clients
     /// <summary>
     /// Proxy implementation for parties
     /// </summary>
+    [ExcludeFromCodeCoverage]
     public class PartiesClient : IPartiesClient
```

Comment:
The PartiesClient should perhaps be rewritten to not consume the SBL bridge API directly but instead just be a proxy for the internal altinn-register API. But this can be made a separate issue for.